test/Microsoft.TemplateEngine.Authoring.TemplateVerifier.IntegrationTests/Snapshots/TemplateEngineSamplesProjectTest.sample02._.verified/sample02/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
```

Comment:
these files have weird naming `._.` should we just define custom name here?

Reply:
I have added custom naming for this test case
please take a look
@vlada-shubina , @JanKrivanek

Reply:
Unfortnuately currently either encoded params, or underscore or custom namers are needed - as Verify requires nonempty MethodName and TypeName and those are then used to name the snapshot. So based on various naming options within TemplateVerifierOptions or in a case that there are no template params, we anyway sometimes need to end up making up some names.
I personaly do not feel the `_` in name should be problematic. It at least explicitly denotes tha fact that no params were specified for instantiation.

tl;dr; Specifying ScenarioName is acceptable workaround.